ansible/configs/ocp-ha-lab/files/hosts_template.3.11.154.j2

```diff
@@ -260,21 +260,23 @@ openshift_grafana_sc_name=glusterfs-storage
 openshift_logging_install_logging={{install_logging}}
 openshift_logging_install_eventrouter={{install_logging}}
 
+openshift_logging_es_pvc_dynamic=true
+openshift_logging_es_pvc_prefix=logging
+openshift_logging_es_pvc_size=20Gi
 {% if install_nfs|bool and not install_glusterfs|bool %}
 openshift_logging_storage_kind=nfs
 openshift_logging_storage_access_modes=['ReadWriteOnce']
 openshift_logging_storage_nfs_directory=/srv/nfs
 openshift_logging_storage_nfs_options='*(rw,root_squash)'
 openshift_logging_storage_volume_name=logging
-openshift_logging_storage_volume_size=10Gi
+openshift_logging_storage_volume_size=20Gi
```

Comment:
Is this still required if you are setting `openshift_logging_es_pvc_size` ?

Reply:
I don't know.  I don't care.  It works. :-)

Reply:
It is not in the docs anywhere, but I guess it is not hurting anything if you want to leave it in there. It will just be confusing for people who are doing the lab part.